src/web/hoc/Color/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function connect(Component) {
```

Comment:
As mentioned earlier, be careful with naming 😛

Reply:
Done :smile: